keyboards/ll3macorn/password/config.h

```diff
@@ -0,0 +1,25 @@
+// Copyright 2021 Ll3macorn (@ll3macorn)
```

Comment:
This board ("password") needs a readme, or to be removed.

Reply:
i will remove it